controllers/replication-controller/persistentvolume_controller_test.go

```diff
@@ -473,9 +473,19 @@ func TestPVReconcileSuite(t *testing.T) {
 	suite.Run(t, testSuite)
 }
 
+func (suite *PVReconcileSuite) TestPVNotFound() {
+	req := suite.getTypicalRequest("non-existent-pv")
+	_, err := suite.reconciler.Reconcile(context.Background(), req)
+	suite.NoError(err, "No error on PV reconcile")
```

Comment:
Test name and assertion do not match. There will be an error returned if PV is not found.